src/ui_ng/lib/src/helm-chart/chart-detail/chart-detail.component.scss

```diff
@@ -13,8 +13,9 @@
     position: absolute;
     top: 0;
```

Comment:
suggest write a mixin and comsuming that. 
Can be:

@mixin absolute-center($width:108px) {
position: absolute;
    top: 0;
    left: 0;
    right:0;
    bottom:0;
    right: 0;
    bottom: 0;
    margin: auto;
    width: $width;
    height: $width;
}